library/src/main/java/com/datatorrent/lib/io/fs/FileSplitterInput.java

```diff
@@ -266,6 +266,7 @@ public TimeBasedDirectoryScanner getScanner()
   public static class TimeBasedDirectoryScanner implements Runnable, Component<Context.OperatorContext>
   {
     private static long DEF_SCAN_INTERVAL_MILLIS = 5000;
+    private static String FILE_BEING_COPIED = "_COPYING_";
```

Comment:
Couple of questions:
1. Why do we have it as independent field and then check it as extension? We can put it as default in ignore regex part right?
2. This is generic class applicable to file systems other than HDFS, it is safe to ignore these files on other file systems? This file may not mean same on other file systems.